index.js

```diff
@@ -0,0 +1,81 @@
+const Asana = require("asana");
+const passport = require("passport");
+const AsanaStrategy = require("passport-asana").Strategy;
+
+const express = require("express");
+const session = require("express-session");
+const app = express();
+const port = 3000;
+
+let client = Asana.ApiClient.instance;
+let token = client.authentications["token"];
+// token.accessToken = process.env.PAT;
```

Comment:
Not sure if this is needed to provided context. Otherwise might want to remove these commented lines